src/certifier.mak

```diff
@@ -83,61 +83,61 @@ $(CL)/certifier.a: $(dobj)
 	$(AR) rcs $(CL)/certifier.a $(dobj)
 
 $(S)/certifier.pb.cc $(I)/certifier.pb.h: $(CP)/certifier.proto
-	$(PROTO) --cpp_out=$(S) --proto_path $(CP) $<
+	$(PROTO) --cpp_out=$(S) --proto_path $(<D) $<
 	mv $(S)/certifier.pb.h $(I)
 
 $(O)/certifier_tests.o: $(S)/certifier_tests.cc $(I)/certifier.pb.h $(I)/certifier.h $(S)/test_support.cc
-	@echo "compiling certifier_tests.cc"
-	$(CC) $(CFLAGS) -c -o $(O)/certifier_tests.o $(S)/certifier_tests.cc
+	@echo "compiling $<"
+	$(CC) $(CFLAGS) -o $@ -c $<
```

Comment:
This is the consistent pattern by which all changes are now done.

Use 

1. `$@` to refer to the target; i.e. `$(O)/certifier_tests.o`
2. `$<` to refer to the 1st pre-requisite file; i.e. `$(S)/certifier_tests.cc`

With these changes, the before / after make rules are now identical.

All make calls further below will be mostly exactly like this, barring an odd-ball case like the make call for `certifier.pb.cc` which needs an extra `-Wno-array-bounds` flag.